upgrade/v2.5/script/upgrade.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+# Set the ENV var for KDD backend
+export DATASTORE_TYPE=kubernetes
+
```

Comment:
`set -e` will exit the job if anything goes wrong, but all 4 migration tasks are independent of each other so we should keep going and try to migrate other resources if we can.
I've updated the PR to do the equivalent of `-o pipefile` (with using ` | tee`